WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -365,11 +375,12 @@ finishedCoinJoin.CoinJoinTask.Status switch
 					&& x.Kitchen.HasIngredients)
 			.ToImmutableDictionary(x => x.WalletName, x => x);
 
-	private IEnumerable<SmartCoin> SelectCandidateCoins(Wallet openedWallet)
+	private IEnumerable<SmartCoin> SelectCandidateCoins(Wallet openedWallet, int bestHeight)
 	{
 		var coins = new CoinsView(openedWallet.Coins
 			.Available()
 			.Confirmed()
+			.Where(x => x.IsInmature(bestHeight))
```

Comment:
```suggestion
			.Where(x => !x.IsInmature(bestHeight))
```
We want to negate this, right? We want coins which are not `Inmature`.